openupgrade_scripts/__manifest__.py

```diff
@@ -6,6 +6,7 @@
         and scripts for migrate Odoo SA modules.""",
     "author": "Odoo SA, Odoo Community Association (OCA),"
     " Therp BV, Opener B.V., GRAP",
+    "maintainers": ["legalsylvain", "StefanRijnhart", "pedrobaeza"],
```

Comment:
I don't see you maintain too much on the script side. I prefer first to put @MiquelRForgeFlow

Reply:
Indeed ! updated.